package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bandwidth/mfa",
-  "version": "1.1.1",
+  "version": "1.2.0",
```

Comment:
I don't think we update the version until you're ready to publish. IOW, updating the version is a separate step that's done at publish time